crates/fuel-core/src/schema/tx.rs

```diff
@@ -132,7 +132,7 @@ impl TxQuery {
             |start: &Option<SortedTxCursor>, direction| {
                 let start = *start;
                 let block_id = start.map(|sorted| sorted.block_height);
-                let all_block_ids = query.compressed_blocks(block_id, direction);
+                let all_block_ids = query.compressed_blocks(block_id.into(), direction);
```

Comment:
Why is this `block_id`? Obviously this pre-existed your PR :)

Reply:
it's the cursor to a paginated query